modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -136,13 +138,25 @@ const QnA: FC<Props> = props => {
   const currentFlow = flows ? flows.find(({ name }) => name === data.redirectFlow) || { nodes: [] } : { nodes: [] }
   const nodeList = (currentFlow.nodes as FlowNode[])?.map(({ name }) => ({ label: name, value: name }))
   const flowsList = flows.map(({ name }) => ({ label: getFlowLabel(name), value: name }))
+  const isNewQna = id.startsWith(NEW_QNA_PREFIX)
 
   return (
     <div className={style.questionWrapper}>
       <div className={style.headerWrapper}>
         <Button minimal small onClick={() => setExpanded(!expanded)} className={style.questionHeader}>
           <div className={style.left}>
             <Icon icon={!expanded ? 'chevron-right' : 'chevron-down'} />{' '}
+            {!isNewQna && (
+              <Tooltip
+                className={cx(style.tag, style.qnaId)}
+                position={Position.BOTTOM}
+                content={lang.tr('module.qna.form.copyIdToClipboard')}
+              >
+                <CopyToClipboard text={id} onCopy={() => toastInfo(lang.tr('module.qna.form.idCopiedToClipboard'))}>
+                  <span onClick={e => e.stopPropagation()}>ID</span>
```

Comment:
is that needed since a span has no click event by default 🤔  `onClick={e => e.stopPropagation()}`

Reply:
Yes, it's required so the click event does not bubble up to an ancestor Div and opens the QnA item view.